backend/src/main/java/com/bakdata/conquery/resources/admin/ui/model/FEPermission.java

```diff
@@ -43,4 +45,9 @@ public static FEPermission from(ConqueryPermission cPermission) {
 		return fePerms;
 	}
 
+	@Override
+	public int compareTo(FEPermission o) {
+		return rawPermission.compareTo(o.rawPermission);
```

Comment:
Sehe nirgends wo du die RawPermission setzt

Reply:
Das passiert in Zeile 37